aten/src/ATen/native/Distance.cpp

```diff
@@ -22,6 +23,35 @@ Tensor pdist(const Tensor& self, const double p) {
   return at::_pdist_forward(self.contiguous(), p);
 }
 
+Tensor cdist(const Tensor& x1, const Tensor& x2, const double p) {
+  AT_CHECK(x1.dim() == 2, "cdist only supports 2D tensors, X1 got: ", x1.dim(), "D");
+  AT_CHECK(at::isFloatingType(x1.type().scalarType()), "cdist only supports floating-point dtypes, X1 got: ", x1.type().scalarType());
+  auto device1 = x1.type().device_type();
+  AT_CHECK(device1 == kCPU || device1 == kCUDA, "cdist only supports CPU and CUDA devices, X1 got: ", device);
```

Comment:
think there's some issue with the merge this (and one case below), should not be `device`.

Reply:
Fixed